src/probnum/statespace/transition.py

```diff
@@ -286,37 +288,54 @@ def smooth_list(self, rv_list, locations, _previous_posterior=None):
         out_rvs.reverse()
         return _RandomVariableList(out_rvs)
 
-    def jointly_sample_list_backward(
-        self, rv_list, locations, _previous_posterior=None
+    # def jointly_sample_list_backward(self, rv_list, t, _previous_posterior=None):
+    #     """Jointly sample from a list of random variables, according to the present
+    #     transition.
+    #
+    #     An explanation of the algorithm can be found in, for instance, this link [1]_.
+    #
+    #     References
+    #     ----------
+    #     .. [1] https://stats.stackexchange.com/questions/376974/how-to-sample-an-unobserved-markov-process-using-the-forward-backward-algorithm
+    #
+    #     Parameters
+    #     ----------
+    #     rv_list : _RandomVariableList
+    #         List of random variables to be sampled from (jointly).
+    #     locations :
+    #         Locations :math:`t` of the random variables in the time-domain. Used for continuous-time transitions.
+    #     _previous_posterior
+    #         Specify a previous posterior to improve linearisation in approximate backward passes.
+    #         Used in iterated smoothing based on posterior linearisation.
+    #
+    #     Returns
+    #     -------
+    #     _RandomVariableList
+    #         List of smoothed random variables.
+    #     """
+    #
+    #     base_measure_samples = np.random.randn(len(t), self.input_dim)
+    #     return self.jointly_transform_base_measure_realization_list_backward(
+    #         rv_list,
+    #         t,
+    #         base_measure_samples=base_measure_samples,
+    #         _previous_posterior=_previous_posterior,
+    #     )
```

Comment:
In the spirit of the randomness in sampling schemes happening in `Posterior.sample()` and the "heavy lifting" being implemented here for convenience's sake, I'd propose to remove this function. The rules of how to transform the samples are captured below, and there is IMO no need for this function to be called here. Thoughts?

Reply:
If you agree, please let me know, and I delete this block before merging

Reply:
I'm not sure if this is still up to date, but in case it is I still agree :)